native-authentication/build.gradle

```diff
@@ -32,7 +32,7 @@ android {
     }
     kotlinOptions {
         jvmTarget = '1.8'
-        freeCompilerArgs += ["-Xopt-in=com.okta.authfoundation.InternalAuthFoundationApi"]
+        freeCompilerArgs += ["-opt-in=com.okta.authfoundation.InternalAuthFoundationApi"]
```

Comment:
This -Xopt-in change is simply for getting rid of some deprecation warnings. It behaves identically otherwise